src/vs/workbench/contrib/terminal/test/browser/xterm/decorationAddon.test.ts

```diff
@@ -19,18 +19,21 @@ import { TestThemeService } from 'vs/platform/theme/test/common/testThemeService
 import { IThemeService } from 'vs/platform/theme/common/themeService';
 import { ILifecycleService } from 'vs/workbench/services/lifecycle/common/lifecycle';
 import { TestLifecycleService } from 'vs/workbench/test/browser/workbenchTestServices';
+import { importAMDNodeModule } from 'vs/amdX';
 
-class TestTerminal extends Terminal {
-	override registerDecoration(decorationOptions: IDecorationOptions): IDecoration | undefined {
-		if (decorationOptions.marker.isDisposed) {
-			return undefined;
+suite('DecorationAddon', async () => {
```

Comment:
@jrieken these newly `async` functions cause test code to be run while other tests are executing. This particular one causes problems and I am changing it in https://github.com/microsoft/vscode/pull/187482.

Reply:
The others do not cause problems at the moment, but I would recommend against doing this.

Reply:
Good catch. I didn't know that an async suite runs in parallel to others.

Reply:
Accurate